### PR TITLE
Hotfix/0.3.1 -> Master

### PR DESCRIPTION
**Pull request by JRogaishio**

### CODE REVIEW

File: src/i18n/es-ES.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ export default {
         volume_slider: 'Control deslizante de volumen',
         rewind_10: 'Rebobinar 10 segundos',
         play: 'Haz clic para reproducir',
+        replay: 'Haga clic para reproducir',
```
Comment (mwarner) [inline src/i18n/es-ES.js:25]: @JRogaishio Should this be "Haga clic para repetir" ?

Comment (JRogaishio) [reply src/i18n/es-ES.js:25]: @mwarner No idea. @tcedge would know since he's a native speaker.

Comment (bigteddo) [reply src/i18n/es-ES.js:25]: hi @JRogaishio  let's change a couple of things on the tranlations 
`      
        volume_slider: 'Control  sonido',
        rewind_10: 'volver 10 segundos atras',
        play: 'Haz click para ver el video ',
        replay: 'haz click para volver ver el video',
`

Comment (JRogaishio) [reply src/i18n/es-ES.js]: @bigteddo Fixed and pushed.

**Review by tcedge — APPROVED** [review]

**Pull request merged**